include/aspect/simulator.h

```diff
@@ -1571,6 +1571,9 @@ namespace aspect
       InitialComposition::Manager<dim>                                        initial_composition_manager;
       InitialTemperature::Manager<dim>                                        initial_temperature_manager;
       const std::unique_ptr<AdiabaticConditions::Interface<dim> >             adiabatic_conditions;
+#if (ASPECT_USE_WORLD_BUILDER)
```

Comment:
We usually write this without parentheses around the variable.
And: In the CMakeLists you define the variable as `ASPECT_WITH_WORLD_BUILDER`, but here you say `ASPECT_USE_WORLD_BUILDER`. Did you check that both the version with and without world builder work?